comms/src/outbound_message_service/message_pool_worker.rs

```diff
@@ -94,42 +106,61 @@ where
             .set_socket_establishment(SocketEstablishment::Connect)
             .establish(&self.inbound_address)
             .map_err(|e| WorkerError::InboundConnectionError(e))?;
-
         loop {
             match inbound_connection.receive(100) {
                 Ok(mut frame_set) => {
                     // This strips off the two ZeroMQ Identity frames introduced by the transmission to the proxy and
                     // from the proxy to this worker
                     let data: FrameSet = frame_set.drain(2..).collect();
                     if let Ok(mut msg) = OutboundMessage::<MessageEnvelope>::try_from(data) {
-                        match self.attempt_message_transmission(&mut msg) {
```

Comment:
I wonder if we shouldn't make the timeout period of line 110 a constant in `type.rs`

Reply:
I will make it a setting in the OMP configuration struct.